tools/update_runnablejobs.py

```diff
@@ -144,44 +151,15 @@ def parse_platform(platform):
     return platform
 
 
-def add_jobs_to_jobpriority(new_data=None, priority=1, timeout=0, set_expired=False):
+def _update_database_with_new_jobs(new_data, set_expiration=False):
+    ''' Return jobs that are new.'''
+    # XXX: jmaher: we might want to change how this method is implemented to do directly query the
+    #              DB by creating a good index. It is probably faster than iterating over rows
```

Comment:
I found that doing hundreds of queries were not that efficient on heroku- in fact I spent about 4 hours trying to get operations to complete before timeout due to queries in a loop.  Possibly you have better ideas for more efficient queries or smarter ones.